client/src/pages/CreatePost.js

```diff
@@ -1,214 +0,0 @@
-import React, { useState } from "react";
-// import axios from "axios";
-import filterOptions from "assets/data/filterOptions";
-import createPostSettings from "assets/data/createPostSettings";
-import CustomModal from "components/CreatePost/CustomModal";
-import RadioGroup from "components/CreatePost/RadioGroup";
```

Comment:
Why is this file being deleted?

Reply:
This is not being used anywhere. I think originally they were thinking of having a whole page to create a new post instead of the modal. But boy did this form break my head so much lol cuz I was changing stuff here and this doesn't get used ANYWHERE :| :(

Reply:
I see. Thank you for explaining.